app/teachingCall/teachingCallStatus/services/teachingCallStatusActionCreators.js

```diff
@@ -214,7 +218,7 @@ teachingCallApp.service('teachingCallStatusActionCreators', function (teachingCa
 				if (teachingCallReceipt.nextContactAtRaw) {
 					var elapsed = elapsedMinutes(teachingCallReceipt.nextContactAtRaw);
 
-					if (elapsed < 10) {
+					if (elapsed < EMAIL_TASK_DELAY) {
```

Comment:
Is this assuming the backend will take 5 minutes? What if the backend is down for 15? It doesn't smell right to do it that way.